src/seedu/addressbook/data/person/Person.java

```diff
@@ -25,6 +28,8 @@ public Person(Name name, Phone phone, Email email, Address address, UniqueTagLis
         this.email = email;
         this.address = address;
         this.tags = new UniqueTagList(tags); // protect internal tags from changes in the arg list
```

Comment:
It's good you add comments. But side comments will more complicate the readability. So I encourage you to describe the purpose of method/constructor at the top.